libcontainer/notify_linux.go

```diff
@@ -35,7 +35,7 @@ func registerMemoryEvent(cgDir string, evName string, arg string) (<-chan struct
 
 	eventControlPath := filepath.Join(cgDir, "cgroup.event_control")
 	data := fmt.Sprintf("%d %d %s", eventfd.Fd(), evFile.Fd(), arg)
-	if err := ioutil.WriteFile(eventControlPath, []byte(data), 0700); err != nil {
+	if err := ioutil.WriteFile(eventControlPath, []byte(data), 0o600); err != nil {
```

Comment:
1. In fact it does not matter what the mode here is, as the file exists (and it can't possibly be created as this is cgroupfs).

2. Might make sense to switch to `fscommon.WriteFile` here.

Reply:
I'll try to switch to `fscommon.WriteFile`.
It's a great way.